exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/ContextFunctions.java

```diff
@@ -64,17 +65,45 @@ public void eval() {
     @Inject DrillBuf buffer;
     @Workspace int currentSchemaBytesLength;
 
+    @Override
     public void setup() {
       final byte[] currentSchemaBytes = contextInfo.getCurrentDefaultSchema().getBytes();
       buffer = buffer.reallocIfNeeded(currentSchemaBytes.length);
       currentSchemaBytesLength= currentSchemaBytes.length;
       buffer.setBytes(0, currentSchemaBytes);
     }
 
+    @Override
     public void eval() {
       out.start = 0;
       out.end = currentSchemaBytesLength;
       out.buffer = buffer;
     }
   }
+
+  /**
+   * Implement "session_id" function. Returns the unique id of the current session.
+   */
+  @FunctionTemplate(name = "session_id", scope = FunctionTemplate.FunctionScope.SIMPLE, isNiladic = true)
```

Comment:
Each drill function is converted to DrillFunctionHolder and though Calcite recognizes current_time/current_date & other context functions (ex: user, current_schema) as niladic, after introduction of isNilladic flag, DrillFunctionHolder would describe such functions as non-niladic, though they are. So we are intentionally hold incorrect info.

Let's say in future I would need to take list of all function holders in Drill (form local function registry) and filter out all niladic functions, since we do have such isNiladic flag I would assume that I can use this flag as filter but since current_time/current_date & other context functions (ex: user, current_schema) have isniladic=false, they would never be filtered out.

Also please add `isNiladic=true` to other niladic context functions [1].

[1] https://github.com/apache/drill/blob/5a4ad2a88331dfe7561ee76fc87e882afe170681/exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/ContextFunctions.java